integrated_channels/moodle/client.py

```diff
@@ -317,7 +317,6 @@ def delete_content_metadata(self, serialized_data):
             querystring=urlencode(course_ids_to_delete))
         return self._post(params, url)
 
-    @moodle_request_wrapper
```

Comment:
I think we need this to ensure a token though. I wonder if there is another way to use the wrapper here to get the right result.

Reply:
I think this is doable. One sec.